src/Cli/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -38,25 +40,32 @@ public static TestCommand FromParseResult(ParseResult result, string[] settings,
             if (settings.Any())
             {
                 // skip '--' and escape every \ to be \\ and every " to be \" to survive the next hop
-                var escaped = settings.Skip(1).Select(s => s.Replace("\\", "\\\\").Replace("\"", "\\\"")).ToArray();
```

Comment:
Have we changed the `.editorconfig` settings for this repo?

Reply:
I don't think we can do it...this repo is used by a lot of different team. I'm simply applying current style where suggested.

Reply:
I also saw the suggestions but they were only set as suggestion so I didn't go with fixing them.